EasyClangComplete.sublime-settings

```diff
@@ -28,6 +28,14 @@
   "cpp_flags" : [
     "-std=c++11"
   ],
+  // Objective-C specific flags. Prepend common_flags for Objective-C files.
+  "objective_c_flags" : [
+    "-std=c11"
+  ],
+  // Objective-C++ specific flags. Prepend common_flags for Objective-C++ files.
+  "objective_cpp_flags" : [
+    "-std=c++11"
```

Comment:
Clang didn't like seeing std c++11 for the Objective C file (not Objective C++). I haven't tried, but I think the same might occur with a test C file instead of C++.